test/com/esotericsoftware/kryo/SerializationCompatTest.java

```diff
@@ -81,7 +83,8 @@
 		int[] versions = new int[] {parseInt(strVersions[0]), parseInt(strVersions[1])};
 		JAVA_VERSION = versions[0] > 1 ? versions[0] : versions[1];
 	}
-	private static final int EXPECTED_DEFAULT_SERIALIZER_COUNT = JAVA_VERSION < 11 ? 57 : 67; // Also change Kryo#defaultSerializers.
+	private static final int EXPECTED_DEFAULT_SERIALIZER_COUNT =
+			JAVA_VERSION < 11 ? UNDER_JAVA_VERSION_11_DEFAULT_SERIALIZER_COUNT : DEFAULT_SERIALIZER_SIZE; // Also change Kryo#defaultSerializers.
```

Comment:
Please inline these constants again. I think they just make the code harder to read.

Reply:
Agree. Readability is important.